### PR TITLE
Document disconnecting members from vc better

### PR DESCRIPTION
**Pull request by bsian03**

### CODE REVIEW

File: lib/Client.js [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ class Client extends EventEmitter {
     * @arg {String} guildID The ID of the guild
     * @arg {String} memberID The ID of the member
     * @arg {Object} options The properties to edit
-    * @arg {String} [options.channelID] The ID of the voice channel to move the member to (must be in voice)
+    * @arg {String?} [options.channelID] The ID of the voice channel to move the user to (must be in voice). Set to `null` to disconnect the user
```
Comment (bsian03) [inline lib/Client.js]:
```suggestion
    * @arg {String?} [options.channelID] The ID of the voice channel to move the member to (must be in voice). Set to `null` to disconnect the member
```

File: lib/structures/Guild.js [hunk 2]
```diff
@@ -556,7 +556,7 @@ class Guild extends Base {
     * Edit a guild member
     * @arg {String} memberID The ID of the member
     * @arg {Object} options The properties to edit
-    * @arg {String} [options.channelID] The ID of the voice channel to move the member to (must be in voice)
+    * @arg {String?} [options.channelID] The ID of the voice channel to move the user to (must be in voice). Set to `null` to disconnect the user
```
Comment (bsian03) [inline lib/structures/Guild.js]:
```suggestion
    * @arg {String?} [options.channelID] The ID of the voice channel to move the member to (must be in voice). Set to `null` to disconnect the member
```

File: lib/structures/Member.js [hunk 3]
```diff
@@ -186,7 +186,7 @@ class Member extends Base {
     /**
     * Edit the guild member
     * @arg {Object} options The properties to edit
-    * @arg {String} [options.channelID] The ID of the voice channel to move the user to (must be in voice)
+    * @arg {String?} [options.channelID] The ID of the voice channel to move the user to (must be in voice). Set to `null` to disconnect the user
```
Comment (bsian03) [inline lib/structures/Member.js]:
```suggestion
    * @arg {String?} [options.channelID] The ID of the voice channel to move the member to (must be in voice). Set to `null` to disconnect the member
```

**Review by curtisf — APPROVED** [review]
Although we might need a general docs update pr later.

**Pull request merged**